mujoco_warp/_src/io.py

```diff
@@ -1715,6 +1719,209 @@ def get_data_into(
   result.sensordata[:] = d.sensordata.numpy()
 
 
+@wp.kernel
```

Comment:
Should we use these kernels in `make_data` and use `wp.empty` instead of `wp.zero` in them?

Reply:
not sure that `reset_data` can currently be called in `make_data` since `reset_data` has a `Model` input whereas `make_data` has a `mujoco.MjModel` input.